include/pybind11/eigen.h

```diff
@@ -10,6 +10,7 @@
 #pragma once
 
 #include "numpy.h"
+#include <numpy/ndarraytypes.h>
```

Comment:
BTW minor formatting nit This uses a different include style `<>` than above `""`.

Reply:
Done.